generators/server/templates/src/main/java/package/config/_FeignConfiguration.java

```diff
@@ -0,0 +1,13 @@
+package <%=packageName%>.config;
+
+import org.springframework.cloud.netflix.feign.EnableFeignClients;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+
+@Configuration
+@Profile("!test")
+@EnableFeignClients(basePackages = "<%=packageName%>.client")
+public class FeignConfiguration {
```

Comment:
no...because

* you can mock a test client this way easier. Running in profile test, no "real" feign clients are constructed
* you can define different non-loadbalanced and non-oauthed feign clients in other packages

this is a fix I applied after working for some weeks with my feign client implementation and stucked on testing this stuff without a faked endpoint for oauth and the other tested service